### DISCUSSION TITLE
Open Source Robotic Arm PAROL 6

### DISCUSSION THREAD

**Discussion by arqfranciscogalicia** [visose/Robots#281] [Q&A] [answered]
Hi Visose

I'm wondering about the possibility of adding an open source robotic arm into the Robots plugin. Currently working on my thesis project at the Politecnico of Milan. Which aims to integrate a robotic arm with a rover. Therefore I turn to this affordable resource, a 3d printable open source robotic arm. https://github.com/PCrnjak/PAROL6-Desktop-robot-arm/tree/main/BOM.
Is this possible?

Than you

**Comment by visose** [comment]
Hi, are you asking me (or someone else) to add support for this robot or are you inquiring if it's technically feasible to add it before you invest the time in attempting it yourself?

**Reply by arqfranciscogalicia** [reply]
Yes, I'm asking you if what I'm suggesting it's possible? I mean to load the Parol 6 Opens source R. Arm into The Robots plug in. If so, how can I do it myself? , Is there an available tutorial to follow the steps? , Of course I appreciate any support from you or anyone for this procedure.

**Reply by visose** [reply]
It's possible, but will likely require making changes in the code. You can check in this pull request how a contributor added support for Fanuc robots: https://github.com/visose/Robots/pull/267

**Comment by visose** [comment, marked as answer]
Closing this as answered. Loading PAROL6 is technically possible, but because it is not one of the supported manufacturers it will likely require code changes, not only an XML/3DM library. PR #267 is a useful example of how a contributor added a new manufacturer path for Fanuc.
